HomeEdu/Scenes/Main/Login/CustomView.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  CustomView.swift
+//  HomeEdu
+//
+//  Created by Nguyen Cuong on 7/31/19.
+//  Copyright © 2019 Nguyen Cuong. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable class CustomView: UIView{
```

Comment:
```suggestion
@IBDesignable class CustomView: UIView {
```
1. sai coding convention
2. đặt tên class rõ nghĩa